src/npc/npc_kill.cpp

```diff
@@ -890,7 +1196,24 @@ void KillNPC(int A, int B)
             }
             else
             {
-                PlaySound(SFX_ShellHit);
+                if(NPCScore[NPC[A].Type] == 1)
+                    PlaySound(SFX_ShellHit);
+                else if(NPCScore[NPC[A].Type] == 2)
+                    PlaySound(SFX_ShellHit);
+                else if(NPCScore[NPC[A].Type] == 3)
+                    PlaySound(SFX_Combo1);
+                else if(NPCScore[NPC[A].Type] == 4)
+                    PlaySound(SFX_Combo2);
+                else if(NPCScore[NPC[A].Type] == 5)
+                    PlaySound(SFX_Combo3);
+                else if(NPCScore[NPC[A].Type] == 6)
+                    PlaySound(SFX_Combo4);
+                else if(NPCScore[NPC[A].Type] == 7)
+                    PlaySound(SFX_Combo5);
+                else if(NPCScore[NPC[A].Type] == 8)
+                    PlaySound(SFX_Combo6);
+                else if(NPCScore[NPC[A].Type] >= 9)
+                    PlaySound(SFX_Combo7);
```

Comment:
A lot of duplicated code pieces...
Please make a function instead of copypasting these code pieces...

Also, you better use the `switch()` instead of `if/else`. Don't repeat Redigit's logic here, we getting to optimise the rest of the code here and around. And so, make your updates being better than Redigit's :)

The function implementation you can put into "sound.cpp" and re-use it at NPC codes around:
```cpp
// At sound.h:
void PlayComboHitSound(int scoreMult);

// At sound.cpp
void PlayComboHitSound(int scoreMult)
{
    if(scoreMult < 2)
        scoreMult = 2; // 2 is minimum possible

    switch(scoreMult)
    {
    case 2: // everything that is 2 or lesser
        PlaySound(SFX_ShellHit);
        break;
    case 3:
        PlaySound(SFX_Combo1);
        break;
    case 4:
        PlaySound(SFX_Combo2);
        break;
    case 5:
        PlaySound(SFX_Combo3);
        break;
    case 6:
        PlaySound(SFX_Combo4);
        break;
    case 7:
        PlaySound(SFX_Combo5);
        break;
    case 8:
        PlaySound(SFX_Combo6);
        break;
    default: // everything that is 9 or bigger
        PlaySound(SFX_Combo7);
        break;
    }
}

// ...

// Usage places:
PlayComboHitSound(NPCScore[NPC[A].Type]);
```